### PR TITLE
Fix issue #280: add setContentType to doGet method

### PR DESCRIPTION
**Pull request by sgilda**

### CODE REVIEW

File: helloworld/src/main/java/org/jboss/as/quickstarts/helloworld/HelloWorldServlet.java [hunk 1]
```diff
@@ -52,6 +52,7 @@
 
    @Override
    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+      resp.setContentType("text/html");
```
Comment (pmuir) [inline helloworld/src/main/java/org/jboss/as/quickstarts/helloworld/HelloWorldServlet.java:55]: Sande, for this one, we should update the guide as well. Do you want to try doing that, so you get familiar with it?

Comment (sgilda) [inline helloworld/src/main/java/org/jboss/as/quickstarts/helloworld/HelloWorldServlet.java:55]: I'd be happy to. I would love to learn how to update the guide. :-)

**Comment by pmuir** [comment]
Were you going to update the guide as well?

**Comment by pmuir** [comment]
@sgilda I suggest a new branch with the changes made - save confusion...

**Comment by sgilda** [comment]
This commit contains the new line of code for both the servlet and for the guide. 

Are you saying you want the change for the guide in a separate branch?

**Comment by pmuir** [comment]
Sorry, I misread! Merged

**Pull request closed without merging**